packages/@webex/internal-plugin-metrics/src/metrics.types.ts

```diff
@@ -102,7 +102,7 @@ export interface ClientEvent {
   options?: SubmitClientEventOptions;
 }
 
-export interface BehavioralEventContext {
+export interface EventContext {
```

Comment:
suggestion: I don't think I like this generalisation `EventContext`.. the problem with this is that the consumer can easily assume this context applies for all metrics, but that's not true... CA metrics don't have this AFAIK.

I'm all in for reusability and common types/structures, but we should be mindful how this affect the understanding of how the plugin works.

Also, if there's a new requirement in the future to support some sort of other metrics which are sent via a diff service than metrics-a, it would get confusing.

Reply:
Fair point, especially since `EVentContext` isn't super precise name either: it doesn't really tells what's kind of context it holds. What about `DeviceContext`? It better defines what's inside and I don't think it would make things more confusing with future requirements.